WhatIsASprint.md

```diff
@@ -0,0 +1,23 @@
+
+# **What is a Sprint?**
+
+
+A **_sprint_** is a set period of time broken out from a project's overall timeline with associated tasks to be completed during that period of time. That list of tasks is referred to as the sprint's <b>Product Backlog</b>.
```

Comment:
Don't use HTML tags. You are writing in markdown, which would then be compiled to HTML. So use MD specific syntax highlighter.